app/config/verify.js

```diff
@@ -0,0 +1,27 @@
+const Joi = require('joi')
+
+// Define config schema
+const schema = Joi.object({
+  totalRetries: Joi.number().integer().default(15),
+  pollingInterval: Joi.number().default(10000), // 10 seconds
```

Comment:
`number().integer()`

Reply:
?

Reply:
Ignore that, thought the comment related to the first line.